config/linker/mold.in

```diff
@@ -0,0 +1 @@
+# mold
```

Comment:
We probably do want to add `depends on CONFIGURE_has_cmake` which will also involve adding something to configure.ac to detect cmake.

Reply:
should probably better a runtime check.

Reply:
So in mold.in add something like

```
config LINKER_MOLD_has_cmake
    def_bool $(success,which cmake)

comment "mold requires cmake"
     depends on LINKER_MOLD_has_cmake
```

?